### ISSUE TITLE
Feature Request: Certificate request manual approvals

### ISSUE THREAD

**Issue by pixdrift** [skx/puppet-summary#69]
I realise this may be going outside the scope/intention of the project as it's not specifically reporting, but I think having the ability to approve pending/manual certificate signing requests similar to how PE provides this feature would be useful.

Specially, in situations where this dashboard is being used to provide an interface to operations teams, having this manual approval would provide the ability to give the task signing responsibility to an operator through dashboard access removing their need to login to shell etc. to approve new servers/certificates.

Basically page structure is 'unsigned certificates' equivalent to
```puppet cert list```

With the ability to sign/reject the cert request which is equivalent to the action
``` puppet cert sign <cert name>```
